assignments/arrays.js

```diff
@@ -75,30 +75,60 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log("Arrays - Challenge 1:");
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}.`);
```

Comment:
This is a good attempt. The downside is that it'll fail if the positions of the items in the array are changed. You could use the array method `find` or `for` loop to get the item whose id is 33, which would be a more stable solution.